### PR TITLE
Gov cleanup + copyright bump

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
Clang formatting based, with a few manual changes

### CODE REVIEW

File: src/governance-object.cpp [hunk 1]
```diff
@@ -463,77 +460,79 @@ bool CGovernanceObject::IsValidLocally(std::string& strError, bool& fMissingMast
...
-                    // this should never happen with CPubKey() as a param
-                    strError = "CheckCollateral critical failure! Masternode: " + strOutpoint;
-                }
-
-                return false;
-            }
+        return true;
+    }
+    case GOVERNANCE_OBJECT_TRIGGER: {
+        if (!fCheckCollateral)
```
Comment (UdjinM6) [inline src/governance-object.cpp]: Let's add braces while at it.

File: src/governance-vote.cpp [hunk 2]
```diff
@@ -11,40 +11,42 @@
...
-        case VOTE_OUTCOME_YES:
-            return "YES"; break;
-        case VOTE_OUTCOME_NO:
-            return "NO"; break;
-        case VOTE_OUTCOME_ABSTAIN:
-            return "ABSTAIN"; break;
+    switch (nOutcome) {
+    case VOTE_OUTCOME_NONE:
+        return "NONE";
+        break;
```
Comment (UdjinM6) [inline src/governance-vote.cpp]: Interesting.. `break;`s make no sense here..

File: src/governance-vote.cpp [hunk 3]
```diff
@@ -11,40 +11,42 @@
...
-            break;
-        case VOTE_SIGNAL_DELETE:
-            strReturn = "DELETE";
-            break;
-        case VOTE_SIGNAL_ENDORSED:
-            strReturn = "ENDORSED";
-            break;
+    switch (nSignal) {
+    case VOTE_SIGNAL_NONE:
+        strReturn = "NONE";
```
Comment (UdjinM6) [inline src/governance-vote.cpp]: Sigh... Two similar functions, two different approaches... And both could actually be refactored/simplified to work the way `ConvertVoteSignal` does it.

File: src/governance.cpp [hunk 4]
```diff
@@ -594,38 +581,34 @@ void CGovernanceManager::DoMaintenance(CConnman& connman)
...
+        if (mapObjects.count(inv.hash) == 1 || mapPostponedObjects.count(inv.hash) == 1) {
             LogPrint("gobject", "CGovernanceManager::ConfirmInventoryRequest already have governance object, returning false\n");
             return false;
         }
-    }
-    break;
-    case MSG_GOVERNANCE_OBJECT_VOTE:
-    {
-        if(cmapVoteToObject.HasKey(inv.hash)) {
+    } break;
```
Comment (UdjinM6) [inline src/governance.cpp]: Let's move this `break;` inside (i.e. smth like s/`} break;`/`break;\n }`/).

File: src/governance.cpp [hunk 5]
```diff
@@ -594,38 +581,34 @@ void CGovernanceManager::DoMaintenance(CConnman& connman)
...
+    case MSG_GOVERNANCE_OBJECT_VOTE: {
+        if (cmapVoteToObject.HasKey(inv.hash)) {
             LogPrint("gobject", "CGovernanceManager::ConfirmInventoryRequest already have governance vote, returning false\n");
             return false;
         }
-    }
-    break;
+    } break;
```
Comment (UdjinM6) [inline src/governance.cpp]: same

File: src/governance.cpp [hunk 6]
```diff
@@ -823,8 +817,7 @@ bool CGovernanceManager::MasternodeRateCheck(const CGovernanceObject& govobj, bo
     LogPrintf("CGovernanceManager::MasternodeRateCheck -- Rate too high: object hash = %s, masternode = %s, object timestamp = %d, rate = %f, max rate = %f\n",
         strHash, masternodeOutpoint.ToStringShort(), nTimestamp, dRate, dMaxRate);
 
-    if (fUpdateFailStatus)
-        it->second.fStatusOK = false;
+    if (fUpdateFailStatus) it->second.fStatusOK = false;
```
Comment (UdjinM6) [inline src/governance.cpp]: I'd rather add braces in original code here.

**Comment by PastaPastaPasta** [comment]
@UdjinM6 resolved comments, specifically take a look at https://github.com/dashpay/dash/pull/2324/commits/41bc3d3d863076cd5e6b10ca7706100963cd51b0 could be a prob in that one

**Comment by PastaPastaPasta** [comment]
Yeah, I expected us not to want that commit in this pr. Dropped that function. Re-review

**Pull request merged**